internal/storage/migrations/0-init-schema.sql

```diff
@@ -17,6 +17,39 @@ create table if not exists connectors (
 );
 create unique index connectors_unique_name on connectors (name);
 
+CREATE OR REPLACE FUNCTION connectors_notify_after_modifications() RETURNS TRIGGER as $$
+BEGIN
+    IF (TG_OP = 'DELETE') THEN
+        PERFORM pg_notify('connectors', 'delete_' || OLD.id);
+        RETURN NULL;
+    ELSIF (TG_OP = 'INSERT') THEN
+        PERFORM pg_notify('connectors', 'insert_' || NEW.id);
+        RETURN NULL;
+    ELSE
+        PERFORM pg_notify('connectors', 'update_' || NEW.id);
+        RETURN NULL;
+    END IF;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER connectors_trigger
+AFTER INSERT OR UPDATE OR DELETE ON connectors FOR EACH ROW
+EXECUTE PROCEDURE connectors_notify_after_modifications();
+
+-- capabilities
+create table if not exists capabilities (
```

Comment:
Wasn't this hardcoded before? Why do we want to store this in the db now?

Reply:
Because we do not want to keep any cache now, because it will imply to call the install method of the plugin on every pod, but we only need once now

Reply:
Ok, but if we have two connectors for the same psp they have the same capabilities right? Why do we need to keep state for connector ID at all, when we could just determine the hard-coded capabilities based on whether the connector is moneycorp or stripe or whatever?

Reply:
@laouji Should be better now !